packages/bits/src/lib/toolbar/toolbar.component.html

```diff
@@ -15,7 +18,11 @@
             </ng-template>
             <nui-divider *ngIf="!last || showMenu" class="flex-shrink-0" isVertical="true" size="extra-small"></nui-divider>
         </ng-template>
-        <nui-menu displayStyle="action" [title]="menuTitle" *ngIf="showMenu">
+        <nui-menu displayStyle="action"
+                  #menuComponent
+                  [title]="menuTitle"
+                  *ngIf="showMenu"
+                  (focusout)="menuComponent.popup.isOpen = false">
```

Comment:
I wonder if this behavior should be built into the menu instead of having to tell the menu "hey, your popup should be closed since you're no longer focused". @dmytrofirsov, @DmytroPutrin what do y'all think?

Reply:
yes, that is hack. Menu has that functional, but not in all cases focusout fires in menu component. Why and exactly when I couldn't answer right now. But I will take a look in the end of the week.

Reply:
thanks 👍

Reply:
If you create a ticket, do you mind adding a note to it that this hack should be removed once it's fixed?